srcpkgs/devour/template

```diff
@@ -0,0 +1,13 @@
+# Template file for 'devour'
+pkgname=devour
+version=11.1
+revision=1
+build_style=gnu-makefile
+hostmakedepends="pkgconf"
```

Comment:
```suggestion
hostmakedepends="pkg-config"
```
most other packages use pkg-config

Reply:
I try to use pkconf in all pkgs I can cause is lighter than pkg-config

Reply:
Please use pkg-config for consistency

Reply:
Hi!
No problem, edited already. Testing change of distfiles to upsream and patches. I'll commit the changes once I have the patches tested.
Thanks!